CMakeLists.txt

```diff
@@ -44,12 +44,12 @@ openpmd_option(HDF5   "Enable HDF5 support"    AUTO)
 openpmd_option(ADIOS1 "Enable ADIOS1 support"  AUTO)
 openpmd_option(ADIOS2 "Enable ADIOS2 support"  OFF)
 # openpmd_option(JSON "Enable JSON support" AUTO)
+option(openPMD_USE_INVASIVE_TESTS "Enable unit tests that modify source code (e.g. C++ keywords)" AUTO)
```

Comment:
can you just use `openpmd_option(INVASIVE_TESTS "Enable unit tests that modify source code (e.g. C++ keywords)"  AUTO)`?

It will handle all we need :)
Sorry, did not see that in the first round

Reply:
I refrained from doing so since it modifies ```openPMD_CONFIG_OPTIONS```, but as you pointed out below that would actually lead to less code.